.nvmrc

```diff
@@ -1 +1 @@
-10.16
+v10.22.0
```

Comment:
I assume nvm is agnostic about whether we include `v` here or not?

Reply:
Yes, both nvm and nfm are agnostic. Maybe it should be fixed to the `v10.22`

I just think that by convention its a little clearer to use the **v** prefix.

Reply:
I can also confirm [the TeamCity build picks it up][tc].

[tc]: https://teamcity.gutools.co.uk/viewLog.html?buildId=566033&buildTypeId=dotcom_master&tab=buildLog&_focus=91&state=103%2C86#_state=103,86